app/models/soda.rb

```diff
@@ -0,0 +1,18 @@
+class Soda < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  belongs_to :last_editor, class_name: 'User'
+  belongs_to :country
+  has_many :photos
+  has_and_belongs_to_many :flavors
```

Comment:
_"We also can create a field: complete_flavor and store a string that is a sum of those tags in the specific order the user entered it if needed" - what would the value of this field be? Where would we use it?_  Not sure. I guess this was my take on compound flavor? I am not sure what exactly we are trying to capture.